core/storage/skill/gae_models_test.py

```diff
@@ -45,11 +46,96 @@ def test_get_deletion_policy_is_not_applicable(self) -> None:
 class SkillModelUnitTest(test_utils.GenericTestBase):
     """Test the SkillModel class."""
 
+    def setUp(self) -> None:
+        super().setUp()
+        self.signup(self.CURRICULUM_ADMIN_EMAIL, self.CURRICULUM_ADMIN_USERNAME)
+        self.user_id_admin = (
+            self.get_user_id_from_email(self.CURRICULUM_ADMIN_EMAIL))
+        self.set_curriculum_admins([self.CURRICULUM_ADMIN_USERNAME])
+
     def test_get_deletion_policy(self) -> None:
         self.assertEqual(
             skill_models.SkillModel.get_deletion_policy(),
             base_models.DELETION_POLICY.NOT_APPLICABLE)
 
+    def test_get_merged_skills(self) -> None:
```

Comment:
Same note applies here and below (about better test names). The test method doesn't need to appear wholesale within the method name. The method name should explain the behaviour that is being tested.

This will also help you write better tests. E.g. in this case the function in question should correctly differentiate between skills that have and haven't been merged. But here you've only tested what is minimal to get coverage.

I'm going to stop reviewing here, I think the same comments apply throughout. If you could address these issues throughout the PR and send it back for review, I'd be happy to take another pass. Thanks!

Reply:
Done